### ISSUE TITLE
Manual installation fail

### ISSUE THREAD

**Issue by greinbold** [wekan/wekan#1155]
## Issue
**Impacted version**: 0.32

**Server Setup Information**:

* Operating System: FreeBSD 11 x64
* Deployment Method(tar/sandstorm/snap/docker/etc /http frontend (nginx, httpd, ...)): tar
* Node Version: 4.8.4
* NPM Version: 2.15.12
* MongoDB Version: 2.6.12
* ROOT_URL environment variable (Is there a subfolder?):

**Installation error**:

`# npm install` produce

```
npm WARN package.json meteor-dev-bundle@0.0.0 No description
npm WARN package.json meteor-dev-bundle@0.0.0 No repository field.
npm WARN package.json meteor-dev-bundle@0.0.0 No README data
npm WARN package.json meteor-dev-bundle@0.0.0 No license field.
npm WARN cannot run in wd meteor-dev-bundle@0.0.0 node npm-rebuild.js (wd=/srv/www/wekan/bundle/programs/server)
```

**Reproduce**:

Type these commands as root:

```
# pkg install node4 npm2 mongodb
# mkdir -p /srv/www/wekan && cd /srv/www/wekan
# fetch https://github.com/wekan/wekan/releases/download/v0.32/wekan-0.32.tar.gz
# tar -xzvf wekan-0.32.tar.gz
# cd bundle/programs/server
# npm install
```

**Comment by xet7** [comment]
@greinbold 

Try these:

1) Docker

https://wiki.freebsd.org/Docker

1a) Directly with Docker:

```
docker run -d --restart=always --name wekan-db mongo:3.2.15

docker run -d --restart=always --name wekan --link "wekan-db:db" -e "MONGO_URL=mongodb://db" -e "ROOT_URL=http://localhost:8080" -p 8080:80 wekanteam/wekan:latest
```

You could also use release tags like wekanteam/wekan:v0.32 because sometimes latest is broken, although currently latest works.

If you want to access Wekan from local network computers, changel localhost to your computer IP address. Then Wekan would be at http://192.168.10.100:8080 or similar. You could also try changing port 8080 to 80 if there's nothing running on 80.

1b) Witch Docker-compose:

https://github.com/docker/compose/issues/4723

https://github.com/wekan/wekan-mongodb

2) Using VirtualBox:

https://wiki.freebsd.org/VirtualBox

https://wekan.xet7.org

**Comment by xet7** [comment]
@greinbold 

Your install from source misses many steps. Please check rebuild-wekan.sh at:
https://github.com/wekan/wekan-maintainer/tree/master/virtualbox

Some of install steps are commented out in that rebuild-wekan.sh script because on VirtualBox I don't install all dependencies every time, as they are already installed.

For running Wekan, in the same repo in run-wekan.sh . You need to change ROOT_URL to your full Wekan URL, like http://192.168.10.100 or http://example.com or http://example.com/wekan .

**Comment by xet7** [comment]
@greinbold 

Have you tried these?

**Comment by greinbold** [comment]
@xet7 

I do not use container like LXC or Docker, only VMs. So I need a clear installation instructions to install manually. I founded the right way to install the version 0.32 on FreeBSD 11.0-RELEASE.

Here you can found my little guide : https://github.com/greinbold/install-wekan

I hope it might be helpful

**Comment by dch** [comment]
As the above URL has disappeared, here's the original content -I'll see if I can get a more current version of wekan & FreeBSD to run based on it, & if so I'll see about making an official port and use a new issue for that.

# Install Wekan v0.32 on FreeBSD 11.0-RELEASE

## Prerequisites

Add user `wekan`

	# adduser

Install prerequisites packages

	# pkg install mongodb node4 npm2

Enable MongoDB

	# sysrc mongod_enable=YES
	# service mongod start

## Installation

As wekan

	$ cd
	$ fetch https://github.com/wekan/wekan/releases/download/v0.32/wekan-0.32.tar.gz
	$ tar -xzpf wekan-0.32.tar.gz

As root

	# chown -R wekan:wekan /home/wekan/bundle

As wekan

	$ cd ~/bundle/programs/server
	$ npm install
	$ npm install fibers

## Run

Considering that the default shell for a new user on FreeBSD is `/bin/sh`, the following ENV variables must be set according the following method before starting of Wekan. These must be adapted according the shell.

	$ MONGO_URL=mongodb://127.0.0.1:27017/wekan; export MONGO_URL
	$ ROOT_URL=https://example.com; export ROOT_URL
	$ MAIL_URL=smtp://user:pass@mailserver.example.com:25/; export MAIL_URL
	$ MAIL_FROM=wekan-admin@example.com; export MAIL_FROM
	$ PORT=8080; export PORT

Now it can be run

	$ cd ~/bundle
	$ node main.js

**Comment by xet7** [comment]
@dch 

Moved to #2662
